src/node_options.cc

```diff
@@ -110,14 +148,15 @@ void NodeOptions::ParseArgs(int* argc,
                             const char*** exec_argv,
                             int* v8_argc,
                             const char*** v8_argv) {
+  // normalize the args
+  // "node -pe" to "node -p -e"
+  char** largv = normalize_args(argc, const_cast<char**>(argv));
```

Comment:
FWIW, seeing how you're adding new code, it's probably better to use a STL container type like `std::vector<...>` instead of juggling malloc/free calls manually.